modules/nf-core/mageck/count/mageck-count.diff

```diff
@@ -55,5 +53,23 @@ Changes in module 'nf-core/mageck/count'
  
  
      cat <<-END_VERSIONS > versions.yml
-
+@@ -40,17 +49,4 @@
+         mageck: \$(mageck -v)
+     END_VERSIONS
+     """
+-    stub:
+-    def args = task.ext.args ?: ''
+-    def prefix = task.ext.prefix ?: "${meta.id}"
+-    def input_file = ("$inputfile".endsWith(".fastq.gz")) ? "--fastq ${inputfile}" : "-k ${inputfile}"
+-    def sample_label = ("$inputfile".endsWith(".fastq.gz") || "$inputfile".endsWith(".fq.gz")) ? "--sample-label ${meta.id}" : ''
+-    """
+-    touch ${prefix}.count.txt
+-    touch ${prefix}.count_normalized.txt
+-    cat <<-END_VERSIONS > versions.yml
+-    "${task.process}":
+-        mageck: \$(mageck -v)
+-    END_VERSIONS
+-    """
```

Comment:
Could we keep the `stub` in our module? It will be useful for testing once all modules have a stub